services/npm/npm-base.js

```diff
@@ -20,6 +24,10 @@ const schema = Joi.object({
       Joi.alternatives(Joi.string(), deprecatedLicenseObjectSchema)
     )
   ),
+  maintainers: Joi.array()
```

Comment:
First thought I had here is about the usage of `maintainers` vs. `collaborators` key. I've seen both used before although I'm honestly not sure what difference they have to npm. However, if we're going to use maintainers here would it make sense to use `maintainers` instead of `collaborators` for the service and path name?

Reply:
The API uses `maintainers`, so in the schema we don't have a choice.

The npm website uses Collaborators:

<img width="364" alt="screen shot 2018-12-08 at 6 41 07 pm" src="https://user-images.githubusercontent.com/1487036/49691829-18956c00-fb19-11e8-97e1-7e3188f9df59.png">

I figure they've updated their terminology but did not modify the API, which is why I went with **collaborators**.

Reply:
Gotcha that makes sense